cli/index.js

```diff
@@ -7,6 +7,7 @@ const { throttling } = require('@octokit/plugin-throttling');
 const { prompt } = require('enquirer');
 const chalk = require('chalk');
 const flatten = require('lodash.flatten');
+const ora = require('ora');
```

Comment:
Note to reviewers:

Since loading the Snyk issues now take a lot longer due to having to crawl the API for more information, I added a spinner to the CLI output so the user can see the progress.